model/src/commonMain/kotlin/ContributorContents.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.confsched2019.model
+
+data class ContributorContents(
+    val contributors: List<Contributor>
+) {
+    companion object {
+        val empty = ContributorContents(emptyList())
```

Comment:
Currently, we use `EMPTY` for the empty object.
We can think about naming. But please use `EMPTY` for now 🙇